ios/PresentationLayer/UIToolkit/Sources/UIToolkit/Utilities/AppTheme.swift

```diff
@@ -44,6 +44,11 @@ public enum AppTheme {
         public static let whisperBackgroundSuccess = Color.green
         public static let whisperBackgroundError = Color.red
         public static let whisperMessage = Color.white
+        
+        // Toast
+        public static let toastSuccessColor = Color(Asset.Colors.success.color)
+        public static let toastErrorColor = Color(Asset.Colors.error.color)
+        public static let toastInfoColor = Color(Asset.Colors.info.color)
```

Comment:
Use only `Asset.Colors.xyz.color` without `Color()` .. otherwise the build is failing

Reply:
Fixed, also I changed the animation.